### PR TITLE
V1.11.1 release prep

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,187 +1,182 @@
...
+     - Enable color by default only if os.Stdout is a TTY ([filecoin-project/lotus#6696](https://github.com/filecoin-project/lotus/pull/6696))
+     - Stop outputing ANSI color on non-TTY ([filecoin-project/lotus#6694](https://github.com/filecoin-project/lotus/pull/6694))
+  - Envvar to disable slash filter ([filecoin-project/lotus#6620](https://github.com/filecoin-project/lotus/pull/6620))
+  - commit batch: AggregateAboveBaseFee config ([filecoin-project/lotus#6650](https://github.com/filecoin-project/lotus/pull/6650))
+  - shed tool to estimate aggregate network fees ([filecoin-project/lotus#6631](https://github.com/filecoin-project/lotus/pull/6631))
+  
+## Bug Fixes
   - Fix padding of deals, which only partially shipped in #5988 ([filecoin-project/lotus#6683](https://github.com/filecoin-project/lotus/pull/6683))
+  - fix deal concurrency test failures by upgrading graphsync and others ([filecoin-project/lotus#6724](https://github.com/filecoin-project/lotus/pull/6724))
+  - PriceListByVersion ([filecoin-project/lotus#6766](https://github.com/filecoin-project/lotus/pull/6766))
```
Comment (ZenGround0) [inline CHANGELOG.md:57]: This fix was reverted so this line should be removed

**Review by arajasek — CHANGES_REQUESTED** [review]
EDIT: Nvm this is only in master
Can we use this opportunity to remove the 1.11.0-rc1 section from the CHANGELOG

**Review by ZenGround0 — COMMENTED** [review]
One small thing then LGTM 🚀

**Review by arajasek — APPROVED** [review]
Address Zen's nit, then ship

**Pull request merged**